modules/raft-api/src/main/java/org/apache/ignite/internal/raft/service/RaftGroupListener.java

```diff
@@ -61,8 +61,14 @@ class ShutdownException extends RuntimeException {
      * Called when a configuration is committed (that is, written to a majority of the group).
      *
      * @param config Configuration that was committed.
+     * @param lastAppliedIndex Last applied index.
+     * @param lastAppliedTerm Last applied term.
      */
-    default void onConfigurationCommitted(CommittedConfiguration config) {
+    default void onConfigurationCommittedWithLastAppliedIndexAndTerm(
```

Comment:
Same thing: I don't think we should rename the method as the parameters already make it clear what index is saved

Reply:
fixed